Required versions of certifi and urllib3 already contain typing, typeshed packages are no longer maintained for them.
--- selenium-4.40.0/pyproject.toml.orig	2000-01-01 01:00:00.000000000 +0100
+++ selenium-4.40.0/pyproject.toml	2026-01-27 17:15:15.147665095 +0100
@@ -30,8 +30,6 @@ dependencies = [
     "trio>=0.31.0,<1.0",
     "trio-websocket>=0.12.2,<1.0",
     "trio-typing>=0.10.0",
-    "types-certifi>=2021.10.8.3",
-    "types-urllib3>=1.26.25.14",
     "typing_extensions>=4.15.0,<5.0",
     "urllib3[socks]>=2.6.3,<3.0",
     "websocket-client>=1.8.0,<2.0",
